dracut/30ignition/test.sh

```diff
@@ -0,0 +1,9 @@
+for i in clevis-encrypt-tpm2 clevis-luks-bind clevis-luks-common-functions clevis-luks-unlock  pwmake tpm2_create clevis-decrypt-tpm2 tpm2_createprimary tpm2_flushcontext tpm2_load tpm2_unseal tpm2_pcrread tpm2_pcrlist clevis-luks-common-functions clevis-decrypt clevis-luks-list luksmeta clevis jose; do printf "#!/bin/sh\nLD_LIBRARY_PATH=/sysusr/usr/lib64 exec /sysusr/usr/bin/"$i" "$@"\n" > "$i"-wrapper; done
```

Comment:
If we want generation we should move this to the module setup.

Reply:
I moved generation and deleted the `test.sh` file.